rurusetto/wiki/templates/wiki/create_ruleset.html

```diff
@@ -48,6 +48,16 @@ <h1>Icon & Logo</h1>
                     </div>
                   </div>
             </div>
+            <div class="fieldWrapper">
+              <div class="mb-3 row">
+                 <label class="col-sm-2 col-form-label form-label">{{ form.dark_icon.label_tag }}</label>
+                 <div class="col-sm-10">
+                   <input type="file" name="dark_icon" accept="image/*" id="id_dark_icon" class="form-control" required>
+                     <p class="form-error raleway-font">{{ form.dark_icon.errors }}</p>
+                     <p class="text-muted raleway-font">An dark icon when that shows on the in-game header in dark theme. Recommend shape is a circle.</p>
```

Comment:
```suggestion
                     <p class="text-muted raleway-font">A light theme icon when user using website in light theme. Recommend shape is a circle.</p>
```
The formhelper information seem a bit wrong.